<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPreferences</class>
 <widget class="QDialog" name="DialogPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>eSpeak NG</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="3">
       <widget class="QPushButton" name="btnEspeakPathBrowse">
        <property name="text">
         <string>&amp;Browse</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lneEspeakPath">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&amp;Path:</string>
        </property>
        <property name="buddy">
         <cstring>lneEspeakPath</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lneEspeakArgs"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&amp;Args:</string>
        </property>
        <property name="buddy">
         <cstring>lneEspeakArgs</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btnEspeakTest">
        <property name="text">
         <string>&amp;Test</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Defaults</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Output Path:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lneDefaultOutputPath"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnDefaultOutputPathBrowse">
        <property name="text">
         <string>B&amp;rowse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>349</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>&amp;Save</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
